test/Microsoft.Sbom.Api.Tests/FormatValidator/SpdxExemplars.cs

```diff
@@ -0,0 +1,306 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom.Api.Tests.FormatValidator;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+internal readonly struct SpdxExemplars
+{
+    public const string JsonSpdx23Exemplar = /*lang=json,strict*/ @"{
```

Comment:
nit: no strong feelings about this, but wondering if it makes more sense to save this as its own spdx file that we use as a test asset instead of keeping it as a string like this.

Reply:
I considered it, but didn't want to have to fiddle with locating the file relative to the path of wherever the test harness executes. Or I guess embedding it as a resource in the test assembly would be another option. I don't feel strongly enough about it to spend the effort, but I will if you have strong feelings.

Reply:
Nope, that's reasonable, works for me!